docs/wp-plugin-check.md

```diff
@@ -0,0 +1,34 @@
+[Back to overview](./README.md)
+
+# wp plugin check
```

Comment:
Can we try using `wp scaffold package-readme` for this so the documentation is automated?

See https://github.com/wp-cli/scaffold-package-command#wp-scaffold-package-readme

Reply:
Sure. I will do research.

Reply:
`wp scaffold package-readme` requires valid package directory as parameter with valid `composer.json`. So as of now we would not be able to use that command in our repo. When we separate our CLI and make it as a package then only we would be able to use this.

Reply:
What updates to composer.json do we need to make?

Reply:
I dug deeper into the scaffold command but faced multiple issues. To generate doc content it used `wp cli cli-dump` commands output but there all commands are there but our `plugin check` is not there. Also location of `README.md` and also file name is not configurable. I am testing in this PR https://github.com/WordPress/plugin-check/pull/356

Reply:
Ah, I see, right, it looks at all the registered commands, but the ones in the plugin are of course only registered when WP is active. Too bad!

Then this should be OK for now.